scripts/us_census/geojsons_low_res/generate_mcf.py

```diff
@@ -34,6 +34,7 @@
 
 class McfGenerator:
     PLACE_TO_IMPORT = 'country/USA'
+    LEVELS_DOWN = 2  # counties
```

Comment:
What about `LEVELS_DOWN = 1`?

Reply:
This value needs to be changed when importing counties (level=2) vs states (level=1). I could make it a command-line argument, but importing states vs counties seems like two different enough operations that they would warrant an explicit change in the code. If you think a command-line argument would be helpful, I can add it though.

Reply:
Let's make it a command-line option.  We want people to be able to use the code without having to modify it.